libs/dh/wholesale/feature-settlement-reports/src/lib/form/dh-wholesale-form.component.ts

```diff
@@ -95,30 +97,35 @@ export class DhWholesaleFormComponent implements AfterViewInit, OnDestroy {
     gridArea: [''],
     period: [
       {
-        value: {
-          start: '',
-          end: '',
-        },
-        disabled: true,
+        start: '',
+        end: '',
       },
     ],
     executionTime: [
       {
-        start: sub(new Date().setHours(0, 0, 0, 0), { days: 10 }).toISOString(),
-        end: new Date().toISOString(),
+        start: '',
+        end: '',
       },
       WattRangeValidators.required(),
     ],
   });
 
   constructor(private fb: FormBuilder) {}
 
-  ngAfterViewInit() {
-    this.filters.valueChanges.pipe(takeUntil(this.destroy$)).subscribe((filters: unknown) => {
-      this.filterChange.emit(filters as SettlementReportsProcessFilters);
-    });
+  private isComplete(filters: SettlementReportFilters) {
```

Comment:
Consideration: We may need a validator for our range inputs to handle these scenarios.

Reply:
Good idea, that sounds like the solution I was looking for 😄 Let's try and implement that in the next PR!